OnlineLibrary.BLL/Services/DataExportService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using OnlineLibrary.Common.Entities;
+using OnlineLibrary.Common.Helpers;
+using OnlineLibrary.BLL.Interfaces;
+using OnlineLibrary.BLL.Model;
+using OnlineLibrary.DAL.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace OnlineLibrary.BLL.Services
+{
+    public class DataExportService : IDataExportService
+    {
+        private readonly IUnitOfWork unitOfWork;
+
+        public DataExportService(IUnitOfWork uow)
+        {
+            unitOfWork = uow;
+        }
+
+        public void WriteCsv(string path, string filename)
+        {
+            List<Book> books = unitOfWork.BookRepository.GetAllBooks();
```

Comment:
**Just think about next**
What if you will need to get all users to csv, or filtered books?